README.md

```diff
@@ -32,6 +32,15 @@ python -m pip install "napari[all]"
 
 If you prefer conda over pip, you can replace the last line with: `conda install -c conda-forge napari`
 
+### containers:
+Docker
+```sh
+docker pull ghcr.io/truatpasteurdotfr/napari:main
+```
+Singularity
+```sh
+singularity exec -B /run oras://ghcr.io/truatpasteurdotfr/napari:latest napari -h
```

Comment:
same here

Reply:
Of course not! I don't expect anyone to use my fork!
You  definitley need to replace ghcr.io/truatpasteurdotfr/napari by ghcr.io/napari/napari as you are building on your CI...
This PR was just meant as a proof of concept "which works for me" :D.

Reply:
does that explain why our build test fails on this PR?

```python-traceback
Traceback (most recent call last):
  File "/home/runner/work/napari/napari/bundle.py", line 312, in <module>
    print('created', bundle())
  File "/home/runner/work/napari/napari/bundle.py", line 288, in bundle
    subprocess.check_call(cmd)
  File "/opt/hostedtoolcache/Python/3.9.9/x64/lib/python3.9/subprocess.py", line 373, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['briefcase', 'build', '--no-docker']' returned non-zero exit status 200.
Error: Process completed with exit code 1.
```

Reply:
> does that explain why our build test fails on this PR?

that looks unrelated, I don't think `bundle.py` should be executing any code in the README

> You definitley need to replace ghcr.io/truatpasteurdotfr/napari by ghcr.io/napari/napari as you are building on your CI...

then that should be stated in the docs.  I guarantee you most people will copy and paste if you don't instruct them (i.e. we can't make assumptions about people's level of understanding here)

Reply:
Well, I can make a newPR with the modified paths if you want.

Reply:
i don't understand... can we just update this PR?

Reply:
I don't know how to update a PR... I will merge the latest push and resubmit.

Reply:
I have modified my fork to reflect the URL changes.